src/ocamlorg_data/job.ml

```diff
@@ -13,7 +13,7 @@ type t =
   
 let all = 
 [
-  { id = 1
+  { id = 2
```

Comment:
@tmattio looks like the `id`s are not stable and they probably should be, if someone shares a link to a job opportunity (e.g. https://v3.ocaml.org/opportunities/1) it now points to the wrong job.

If `jobs.yml` is an append-only list then we can just number them in order and add a failed to indicate the opening is no longer available?

Reply:
> If jobs.yml is an append-only list then we can just number them in order and add a field to indicate the opening is no longer available?

That's how it's supposed to work, will have a look 🙂

Reply:
Oh I see what happened there! It was expected that users will add job posts at the top of the list: we reverse the list to populate the IDs, and reverse it back once populated.

I doubt that there are any links to the job posts at the moment, so I think we can keep the IDs currently in `main`, and maybe add a comment to the YAML file to ask users to add the posts at the top of the list.